gensim/matutils.py

```diff
@@ -70,7 +70,7 @@ def argsort(x, topn=None):
             return numpy.argsort(x)[:topn]
         # numpy >= 1.8 has a fast partial argsort, use that!
         biggest = numpy.argpartition(x, topn)[:topn]
-        return biggest.take(numpy.argsort(x.take(biggest)))  # resort topn into order
```

Comment:
This doesn't look right -- the sign is already changed 4 lines above. Are you sure this works?